src/backend/marsha/core/lti/__init__.py

```diff
@@ -107,7 +112,28 @@ def verify(self):
                 )
             )
 
-        return consumer_site
+        self._consumer_site = consumer_site
+        return True
+
+    def get_consumer_site(self):
```

Comment:
if you don't modify verify and let it return the consumer_site we probably don't need this method?

Reply:
I think `verify` should return nothing. It verifies the request, that's all.